salt/orchestrate/aws/create_vpc.sls

```diff
@@ -23,6 +23,9 @@ create_{{ ENVIRONMENT }}_vpc:
     - tags:
         Name: {{ VPC_NAME }}
         business_unit: {{ BUSINESS_UNIT }}
+        Department: {{ BUSINESS_UNIT }}
+        OU: {{ BUSINESS_UNIT }}
+        Environment: {{ ENVIRONMENT }}
```

Comment:
If we're now adding all those tags to a resource, is there some ways to bundle it up into a list and add it to env_settings and just have one statement to iterate through it?

Reply:
I think that it would make sense for us to encapsulate the tagging of resources into a reusable component (jinja variable, macro, python lib, etc.) but that is something for us to think about as a separate task.